build.gradle

```diff
@@ -25,7 +23,6 @@ buildscript {
     }
 
     dependencies {
-        classpath 'org.kordamp.gradle:clirr-gradle-plugin:0.2.2'
```

Comment:
I hate to lose clirr.... What's broken about it and can it be fixed?

Reply:
It hasn't worked for a long, long time.  Keeps giving errors about missing classes and the like.  I've tried again and again to compare to the driver's configuration but haven't been able to spot what's different.